package.json

```diff
@@ -468,10 +468,20 @@
         "confluent.preview.enableProduceMessages": {
           "type": "boolean",
           "default": false,
-          "markdownDescription": "Enable message production for Kafka topics.\n\n---\n\n⚠️ **WARNING**: This setting may allow you to produce messages without schemas to topics that expect them, which can lead to data loss or corruption.",
+          "markdownDescription": "Enable message production for Kafka topics.",
           "tags": [
             "preview"
           ]
+        },
+        "confluent.topic.produceMessages.schemas.useTopicNameStrategy": {
+          "type": "boolean",
+          "default": true,
+          "markdownDescription": "Use [TopicNameStrategy](https://docs.confluent.io/platform/current/schema-registry/fundamentals/serdes-develop/index.html#overview) for schema subject name when producing messages to Kafka topics. Disabling this will require selecting the subject name strategy to use."
+        },
+        "confluent.topic.produceMessages.schemas.allowOlderVersions": {
```

Comment:
I'm not entirely convinced we even need this here. It would probably be better to always enable but warn on old version selection. But given time I'd rather have this better than neither and I don't feel strongly about it.